auto_nag/scripts/assignee_no_login.py

```diff
@@ -73,9 +79,6 @@ def get_bz_params(self, date):
             "f1": "assignee_last_login",
             "o1": "lessthan",
             "v1": start_date,
-            "f2": "days_elapsed",
-            "o2": "lessthan",
-            "v2": self.last_activity_months * 30,
```

Comment:
I guess removing this limitation will result of having a long list of bugs to be unassigned at once (maybe hundreds); especially for the first run. 

Do we want to do this on multiple patches? We could use `"limit": 100` in params to mitigate this.

Reply:
Yeah, the max_ni of 14 and the weekly run should help with that (each person will get a maximum of 14 needinfos per week)

Reply:
Using `"limit": 100` will ensure that we are not targeting too many bugs in the same week, less anyone for team wid scope (some bugs could have a snowball impact). But I agree max_ni of 14 could be enough.

Reply:
Tested in dry run mode, I think we are fine this way.